packages/lit-virtualizer/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@lit-labs/virtualizer",
-    "version": "0.7.0-pre.2",
+    "version": "0.8.0",
```

Comment:
This isn't a breaking change, so I think we can stick with 0.7. Let's do 0.7-pre.3

Reply:
@hunterloftis could we get this changed?